src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -251,12 +251,23 @@ void GcsServer::InitGcsActorManager(const GcsInitData &gcs_init_data) {
     return std::make_shared<rpc::CoreWorkerClient>(address, client_call_manager_);
   };
 
+  std::function<bool(std::shared_ptr<GcsActor>)> release_resources =
```

Comment:
don't we already have a default one?

Reply:
Fixed. Now this function is declared here with `nullptr` default value.